common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -187,7 +195,7 @@ private bool HasOneTimeSetup()
         }
 
         [OneTimeTearDown]
-        public void OneTimeCleanupResourceGroups()
+        public virtual void OneTimeCleanupResourceGroups()
```

Comment:
do we need these are virtual?

Reply:
Yeah... we do not need this. Will remove this